### DISCUSSION TITLE
setting font size for flashcards

### DISCUSSION THREAD

**Discussion by huangziwei** [st3v3nmw/obsidian-spaced-repetition#406] [Q&A]
Hello. 

Is there a way to set font size? I found the font in flashcards is too small.

**Comment by ikuygithub** [comment]
same here.
no way to enlarge font on flashcard?
I wonder how everybody gets used to that micro font!!

**Comment by st3v3nmw** [comment]
Hi @huangziwei & @ikuygithub,
Someone has worked on a fix for this, this change should be released soon. Tracking it in #497
